assets/questions/rle.json

```diff
@@ -92,7 +94,7 @@
         "acquired_skills": ["String Manipulation", "Sets", "Arrays", "Maps"],
         "input_function": null,
         "output_function": null,
-        "main_function": "sample.DecodeEncodedString",
+        "main_function": "DecodeEncodedString",
```

Comment:
Which other schema? This DecodeEncodedString is something that _I'm_ writing, not something the student should be writing.